Description: Revert module version zeroing
 Trying to revert the zeroing since a crash seems to happen without  
 non-zero version number. (LP: #1178620)

diff --git a/.qmake.conf b/.qmake.conf
index 8840bd9..3c4fa5e 100644
--- a/.qmake.conf
+++ b/.qmake.conf
@@ -1,3 +1,3 @@
 load(qt_build_config)
 
-MODULE_VERSION = 0.0.0
+MODULE_VERSION = 5.0.0

